app/src/main/java/com/arygm/quickfix/ui/profile/becomeWorker/UpgradeToWorkerScreen.kt

```diff
@@ -0,0 +1,99 @@
+package com.arygm.quickfix.ui.profile.becomeWorker
+
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.ArrowBack
+import androidx.compose.material3.*
+import androidx.compose.material3.MaterialTheme.colorScheme
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.platform.testTag // Import pour les testTag
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.testTag
+import androidx.compose.ui.unit.dp
+import com.arygm.quickfix.model.account.AccountViewModel
+import com.arygm.quickfix.model.account.LoggedInAccountViewModel
+import com.arygm.quickfix.model.profile.ProfileViewModel
+import com.arygm.quickfix.ressources.C
+import com.arygm.quickfix.ui.navigation.NavigationActions
+import com.arygm.quickfix.ui.profile.becomeWorker.views.personal.PersonalInfoScreen
+import com.arygm.quickfix.ui.theme.poppinsTypography
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun BusinessScreen(
+    navigationActions: NavigationActions,
+    accountViewModel: AccountViewModel,
+    workerProfileViewModel: ProfileViewModel,
+    loggedInAccountViewModel: LoggedInAccountViewModel
+) {
+  val pagerState = rememberPagerState(pageCount = { 3 })
+  val focusManager = LocalFocusManager.current
+  val displayName = remember { mutableStateOf("") }
+  val description = remember { mutableStateOf("") }
+  val imagePathPP = remember { mutableStateOf("") }
+  val imagePathBP = remember { mutableStateOf("") }
+  var displayNameError by remember { mutableStateOf(false) }
+  var descriptionError by remember { mutableStateOf(false) }
+  Scaffold(
+      modifier =
+          Modifier.pointerInput(Unit) { detectTapGestures(onTap = { focusManager.clearFocus() }) }
+              .semantics { testTag = C.Tag.upgradeToWorkerScaffold },
+      topBar = {
+        TopAppBar(
+            title = {
+              Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
+                Text(
+                    "Business Account",
```

Comment:
You did a good job,just a quick remark consider in the future moving these into a centralized resource file or constants to facilitate localization and maintenance. you can relosve this